app/build.gradle

```diff
@@ -62,6 +62,7 @@ dependencies {
 
     // Google Maps Services
     implementation 'com.google.android.gms:play-services-maps:16.0.0'
+    implementation 'org.jetbrains:annotations-java5:15.0'
```

Comment:
could you add this libraries usage into the reuse statement

Reply:
✅